src/transformers/models/llama/convert_llama_weights_to_hf.py

```diff
@@ -91,6 +91,7 @@ def write_model(model_path, input_base_path, model_size, tokenizer_path=None, sa
 
     params = read_json(os.path.join(input_base_path, "params.json"))
     num_shards = NUM_SHARDS[model_size]
+    params = params.get("model", params)
```

Comment:
Reason: Model parameters are nested in the Llama Guard configuration file: https://huggingface.co/meta-llama/LlamaGuard-7b/blob/main/params.json#L67, whereas they weren't in previous releases.